src/runtimes/node/in_source_config/index.ts

```diff
@@ -8,22 +8,27 @@ import { getRoutesFromPath, Route } from '../../../utils/routes.js'
 import { RUNTIME } from '../../runtime.js'
 import { NODE_BUNDLER } from '../bundlers/types.js'
 import { createBindingsMethod } from '../parser/bindings.js'
-import { getExports } from '../parser/exports.js'
+import { traverseNodes } from '../parser/exports.js'
 import { getImports } from '../parser/imports.js'
 import { safelyParseSource, safelyReadSource } from '../parser/index.js'
+import type { ModuleFormat } from '../utils/module_format.js'
 
 import { parse as parseSchedule } from './properties/schedule.js'
 
 export const IN_SOURCE_CONFIG_MODULE = '@netlify/functions'
 
 export type ISCValues = {
-  invocationMode?: InvocationMode
   routes?: Route[]
-  runtimeAPIVersion?: number
   schedule?: string
   methods?: string[]
 }
 
+export interface StaticAnalysisResult extends ISCValues {
```

Comment:
We were using the term "ISC" to contain properties that are not part of the in-source configuration spec, like `invocationMode` or `runtimeAPIVersion`. They were grouped together because we extract both those properties _and_ the ISC properties using the same static analysis step, but functionally they are different things.

So now we have a `ISCValues` type that holds just the in-source configuration properties, and a `StaticAnalysisResult` that holds those plus some additional metadata properties we'll extract using the static analysis mechanism.

A big part of this diff is just renaming ISC-related terms to a more generic "static analysis result".